### ISSUE TITLE
Property Filtering: Zoning

### ISSUE THREAD

**Issue by soskin** [CodeForPhilly/clean-and-green-philly#1058]
**Is your feature request related to a problem? Please describe.**

Under Find Properties, when further filtering on Zoning designation, the various levels are too complex for intended low-capacity user.  ie. CA-1, CA-2, CMX-1, CMX-2, CMX-3 ... 

**Describe the solution you'd like**
The Zoning designation per property is what it is, fact. 
However, filtering could be easier, rolling up to a general zoning designation.  
Individuals/low cap care less / don't necessarily differentiate between CMX-1,2,3,4 etc. vs. simply CMX. 
All residential properties toll up to RSD, RSA, RTA, RM, RMX, CA, CMX, IRMX, ICMX, I, SP  (remove all the sub-categories, for purposes of filtering, only). 

Should also be able to pick more than one. 

**Describe alternatives you've considered**
Can leave as is ...
Roll some up and not others ... 
Attached the guide, somewhere on the site, if you think helpful. 

**Additional context**

https://www.phila.gov/media/20220909084529/ZONING-QUICK-GUIDE_PCPC_9_9_22.pdf

**Comment by DonovanAndrews300** [comment]
I agree with the complexity of the zoning filter for the users. I would like to tackle this one.

**Comment by soskin** [comment]
Great! LMK if you need anything.
@CodeWritingCow can you assign?

**Comment by nlebovits** [comment]
@DonovanAndrews300 assigned! Thanks for tackling. Implementation preference would be to allow users to also start by typing the zoning code and have it filter that way, too. (e.g., if you started typing "RS...", it automatically narrows the list to only those starting with "RS"). We could then reuse this filtering component for the RCO and Neighborhood filters, and possibly others.

**Comment by DonovanAndrews300** [comment]
Got it! So remove subzones from the filter options, add a reusable component for filtering, allow multiple zones to be selected simultaneously, and enable users to type to narrow down the filter options.

**Comment by soskin** [comment]
This is great. 
A compact effective list! Many thx.

**Comment by CodeWritingCow** [comment]
Hi @DonovanAndrews300 I had to revert this ticket's PR #1064 due to a bug. When you get a chance, can you take a look? Thank you!

**Comment by DonovanAndrews300** [comment]
@CodeWritingCow Thanks for catching that! I’ll dig in and see what’s going on. I’ll keep you posted once I’ve sorted it out.

**Comment by soskin** [comment]
@DonovanAndrews300 @CodeWritingCow Great work on this!!
